package.json

```diff
@@ -40,6 +40,9 @@
     "publish": "lerna publish"
   },
   "dependencies": {
+    "@carbon/icons-react": "^11.52.0",
+    "@carbon/react": "^1.73.0",
+    "@carbon/styles": "^1.72.0",
```

Comment:
I think we need to move these dependencies to `devDependencies` and also write them down in the `peerDependencies` section as well, so this library will use what's provided in the host app to avoid some of the version conflicts.

I also think we can remove `@patternfly/*` dependencies from both `devDependencies` and `peerDependencies`